src/CorrugatedIron.Tests/Models/MapReduce/RiakMapReduceTests.cs

```diff
@@ -38,6 +40,9 @@ public class RiakMapReduceTests
         private const string ComplexMrJobWithFilterText =
             @"{""inputs"":{""bucket"":""animals"",""key_filters"":[[""matches"",""spider""]]},""query"":[{""map"":{""language"":""javascript"",""source"":""function(o) { return [1]; }"",""keep"":false}},{""reduce"":{""language"":""javascript"",""name"":""Riak.reduceSum"",""keep"":true}}]}";
 
+        private const string ComplexMrJobWithTypeAndFilterText =
+            @"{""inputs"":{""bucket"":[""zoo"",""animals""],""key_filters"":[[""matches"",""spider""]]},""query"":[{""map"":{""language"":""javascript"",""source"":""function(o) { return [1]; }"",""keep"":false}},{""reduce"":{""language"":""javascript"",""name"":""Riak.reduceSum"",""keep"":true}}]}";
```

Comment:
Yeah, there's no great way between the options. I would hate the resources file for debugging tests though.